File: Diagram/DiagramFilterCriteria.vb
﻿'************************************************************************************
' STSimStockFlow: A .NET library for simulating stocks and flows
'
' Copyright © 2007-2017 Apex Resource Management Solution Ltd. (ApexRMS). All rights reserved.
'
'************************************************************************************

Class DiagramFilterCriteria

    Private m_FlowTypes As New Dictionary(Of Integer, Boolean)

    Public ReadOnly Property FlowTypes As Dictionary(Of Integer, Boolean)
        Get
            Return Me.m_FlowTypes
        End Get
    End Property

End Class

File: src/Diagram/DiagramFilterCriteria.cs
// STSimStockFlow: A SyncroSim Package for the ST-Sim Stocks and Flows Add-In.
// Copyright © 2007-2019 Apex Resource Management Solution Ltd. (ApexRMS). All rights reserved.

using System.Collections.Generic;

namespace SyncroSim.STSimStockFlow
{
	internal class DiagramFilterCriteria
	{
		private Dictionary<int, bool> m_FlowTypes = new Dictionary<int, bool>();

		public Dictionary<int, bool> FlowTypes
		{
			get
			{
				return this.m_FlowTypes;
			}
		}
	}
}